multi-server-install-planning.adoc

```diff
@@ -220,13 +222,22 @@ resides on Zimbra Mailbox server) using the Zimbra Mailbox Ports.
 |IMAPS Backend (if Proxy configured) |7993
 |========================
 
+[cols=",",options="header",]
+|========================
+2+|Zimbra IMAPD Ports (Internal to {product-name})
```

Comment:
hmm... sure seems like we shouldn't have the word Zimbra here... but then we still haven't replaced all of those with {variable} either.